src/metal.cs

```diff
@@ -3342,9 +3646,29 @@ interface MTLComputePipelineDescriptor : NSCopying {
 		[Export ("buffers")]
 		MTLPipelineBufferDescriptorArray Buffers { get; }
 
-		[NoMac, iOS (13, 0), TV (13,0)]
+		[Mac (11,0), iOS (13, 0), TV (13,0)]
 		[Export ("supportIndirectCommandBuffers")]
 		bool SupportIndirectCommandBuffers { get; set; }
+
+		[Mac (11, 0), iOS (14, 0), TV (14,0)]
+		[NullAllowed, Export ("insertLibraries", ArgumentSemantic.Copy)]
+		IMTLDynamicLibrary[] InsertLibraries { get; set; }
+
+		[Mac (11, 0), iOS (14, 0), TV (14,0)]
+		[NullAllowed, Export ("binaryArchives", ArgumentSemantic.Copy)]
+		IMTLBinaryArchive[] BinaryArchives { get; set; }
+
+		[Mac (11, 0), iOS (14, 0), TV (14,0)]
+		[NullAllowed, Export ("linkedFunctions", ArgumentSemantic.Copy)]
+		MTLLinkedFunctions LinkedFunctions { get; set; }
+
+		[Mac (11, 0), iOS (14, 0), NoTV] // fails introspection on tv on simulator and device
```

Comment:
Does it work in an Xcode project?

Reply:
This is still not answered.

Reply:
Wrong comment, removing, is not supported on tvOS.